### ISSUE TITLE
No login possible on latest version

### ISSUE THREAD

**Issue by Pabsilon** [segalion/securitasdirect#22]
Hi!

After installing the lastest version from HACS, and restarting home assistant, I got the following logs:

> 2020-11-12 19:17:26 DEBUG (SyncWorker_8) [custom_components.securitas_direct.securitas] OrderedDict([('PET', OrderedDict([('RES', 'ERROR'), ('ERR', '60161'), ('MSG', 'Ahora mismo solo tienes acceso a la nueva versión de My Verisure, haz clic en aceptar para acceder a la nueva versión.'), ('BLOQ', OrderedDict([('@remotereqactive', '1'), ('#text', 'Estamos mejorando nuestros servicios. Por favor intentelo de nuevo mas tarde. Gracias por confiar en Securitas Direct')]))]))])
> 2020-11-12 19:17:26 ERROR (SyncWorker_8) [custom_components.securitas_direct.securitas] Ahora mismo solo tienes acceso a la nueva versión de My Verisure, haz clic en aceptar para acceder a la nueva versión.
> 2020-11-12 19:17:26 DEBUG (SyncWorker_8) [custom_components.securitas_direct] Log in Securitas: False
> 2020-11-12 19:17:26 ERROR (SyncWorker_8) [custom_components.securitas_direct] Could not log in to Securitas: False

So it talks about a newer version of the app (that I've been using on the phone since the nov 4th api change). Until I updated earlier today, I still managed to get the alarm states in home assistant, but I wasn't able to change them. Now it just won't log in.

My configuration is as follows:
`securitas_direct:
  username: username
  password: password
  #code: 1234
  country: ES`
I tried with and without the code commented and had the same errors.
Could this be tied to a user by user migration on verisure's side? 

Thank you for your time and your great work! Keep it up!

**Comment by segalion** [comment]
See this post:

https://community.home-assistant.io/t/securitas-direct-verisure-eu-custom-component-testers-wanted/70239/246?u=segalion

You can try to complain to company to continue using old app, and this component will work again.

Please tell me if they make the change.

**Comment by Pabsilon** [comment]
Great! I didn't think about looking on the HA thread.

I called Securitas and asked for a migration to the old app, alluding that one of our phones doesn't support the new version, and someone installed the new one prompting the migration. Apparently it's a bit common, since they had no objections in doing the migration to the old system.

They've told me that in around 3 working days the migration to the old app will be done. I'll report back if that gets me working again.

Thank you!

**Comment by aladin2000** [comment]
it seems that your last code is correct regarding the new parameters ordering status 
 
     python3 ./securitas.py FR USER PASSWD  PERI lang=fr 

it is OK now 

thanks so lot !

**Comment by Vartkat** [comment]
> it seems that your last code is correct regarding the new parameters ordering status
> 
> ```
>  python3 ./securitas.py FR USER PASSWD  PERI lang=fr 
> ```
> 
> it is OK now
> 
> thanks so lot !

Can you elaborate a bit please so that we can follow a few instructions to get it working again.

**Comment by aladin2000** [comment]
I tested the New code and  I changed the original way to use thé python script
With old api , I used to do  the call : 

python3 ./securitas.py user passwd Command FR FR 

Now i changed it to follow New order in parm

 python3 ./securitas.py FR user passwd Command  lang=FR

Hope this help

17 nov. 2020 14:45:14 Vartkat <notifications@github.com>:

**Comment by Pabsilon** [comment]
Good news!
After numerous calls, they rolled my account back to the 'old' app. And it works flawlessly.

So... You should add a warning on the README, so that people avoid the new app like the plague.

**Comment by jmservera** [comment]
It happened the same to me, but I noticed that the previous commit #9ba8878eb6 is partially working; the login and status read work but I cannot arm or disarm. The command line from https://github.com/Cebeerre/VerisureAPIClient is also working after migrating to the new app, so, there is a way to make it work, but I don't have the time right now to create a fix. If I have the time in a few weeks I will try to fix it, but by now I think I will stay in an older build of the component because I'm more interested in the events than in the operation by now.

**Comment by Cebeerre** [comment]
> It happened the same to me, but I noticed that the previous commit #9ba8878eb6 is partially working; the login and status read work but I cannot arm or disarm. The command line from https://github.com/Cebeerre/VerisureAPIClient is also working after migrating to the new app, so, there is a way to make it work, but I don't have the time right now to create a fix. If I have the time in a few weeks I will try to fix it, but by now I think I will stay in an older build of the component because I'm more interested in the events than in the operation by now.

@jmservera I'm quite surprised it still works ... my understanding was that in the new version of the App they were using a new API ...

**Comment by KoolLSL** [comment]
I am using the recent nragon commit ahead 312ec25
 https://github.com/nragon/securitasdirect with the new Securitas Direct app, and so far the HA alarm-panel or automations are working well with it.  I had to add installation: number after user, password, in the configuration.yaml, country PT

**Comment by jmservera** [comment]
> 
> 
> I am using the recent nragon commit ahead [312ec25](https://github.com/segalion/securitasdirect/commit/312ec25a118a8e835e4ff490631fba8c818d1187)
> https://github.com/nragon/securitasdirect with the new Securitas Direct app, and so far the HA alarm-panel or automations are working well with it. I had to add installation: number after user, password, in the configuration.yaml, country PT

Yep! I can confirm it works seamesly in ES as well, but the fork needs still some work because now the integration shows up as an "unnamed device".

**Comment by KoolLSL** [comment]
For those still asking, the commit ahead 96fd7a5 on 18 Dec 2020 from https://github.com/nragon/securitasdirect is working very well once you installed the new Verisure app on your phone. The "unnamed device" glitch has been fixed, it shows now "alarm_control_panel.securitas_xxxxx".  As the component is not yet published here, I have replaced manually the 4 files files in /config/custom_components/securitas_direct/ with the nragon's ones. Also add installation: number in your configuration.yaml

**Comment by aladin2000** [comment]
**Also add installation: number in your configuration.yaml**

I probably am wrong, but using that parameter conduct to a "**unknown option" for installation**

**Comment by aladin2000** [comment]
> For those still asking, the commit ahead [96fd7a5](https://github.com/segalion/securitasdirect/commit/96fd7a513f7953c239fd892cf8f29d3697c6aa26) on 18 Dec 2020 from https://github.com/nragon/securitasdirect is working very well once you installed the new Verisure app on your phone. The "unnamed device" glitch has been fixed, it shows now "alarm_control_panel.securitas_xxxxx". As the component is not yet published here, I have replaced manually the 4 files files in /config/custom_components/securitas_direct/ with the nragon's ones. Also add installation: number in your configuration.yaml

what is the new way for  cli mode with parameter "application" ?
  ``   python3 ./securitas.py FR user passwd Command  lang=FR  **_application_ number_**  ?
  ``

**Comment by guerrerotook** [comment]
Checkout this last issue #37
